requirements.txt

```diff
@@ -1,2 +1,2 @@
-setuptools==49.2.1
-pdoc3==0.9.1
```

Comment:
Aren't these `--dev` requirements (in Pipenv terminology)?

Reply:
This could indeed go into a `requirements_dev.txt` file, it's actually more a `deploy` this, not development, since you don't actually need it. I will see to change that.